dashboard/src/components/commons/Header.js

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Header = ({ style, logo }) => (
-  <nav className="navbar navbar-default navbar-fixed-top navbar-custom">
+const Header = ({ appStyle, projectStyle, appLogo, projectLogo }) => (
+  <header className="Header">
     <div className="container-fluid">
-      <div className="navbar-logo-container">
-        <img style={style} src={logo.url} alt="Application Logo" />
+      <div className="d-flex align-items-center justify-content-between Header__row">
+        <div className="Header__row__left-part d-flex align-items-center">
+          <div className="Header__row__logo">
+            <a href="">
```

Comment:
How about removing empty anchor tags with `span` or `div` since they aren't pointing anywhere and are probably used just for styling purposes?
```suggestion
            <div>
              <img style={appStyle} src={appLogo.url} alt="Application Logo" />
            </div>
```